modules/sugar/src/test/ts/module/ephox/sugar/test/WithHelpers.ts

```diff
@@ -3,6 +3,8 @@ import { document, Element as DomElement, HTMLIFrameElement, ShadowRoot, Window
 import * as Insert from 'ephox/sugar/api/dom/Insert';
 import * as Body from 'ephox/sugar/api/node/Body';
 import * as Remove from 'ephox/sugar/api/dom/Remove';
+import * as ShadowDom from 'ephox/sugar/api/node/ShadowDom';
+import { Attr } from '@ephox/sugar';
```

Comment:
This import should be `import * as Attr from 'ephox/sugar/api/properties/Attr';`

Reply:
Well, that's embarrassing. Fixed.